pcdsdevices/spectrometer.py

```diff
@@ -361,3 +362,45 @@ class HXRSpectrometer(BaseInterface, GroupDevice):
     removed = False
     transmission = 1
     SUB_STATE = 'state'
+
+
+class Gen1VonHamosCrystal(BaseInterface, GroupDevice):
+    """Pitch, yaw, and translation motors for control of a single crystal."""
+
+    tab_component_names = True
+
+    pitch = FCpt(EpicsMotorInterface, '{prefix}{_pitchAxis}', kind='normal')
+    yaw = FCpt(EpicsMotorInterface, '{prefix}{_yawAxis}'  , kind='normal')
+    trans = FCpt(EpicsMotorInterface, '{prefix}{_transAxis}', kind='normal')
+
+    def __init__(self, prefix, pitchAxis, yawAxis, transAxis, **kwargs):
```

Comment:
Small quibble here, these variable names violate the pep8 conventions
https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
Hi Zach, no problem. Changed them to be PEP 8 – Style Guide for Python Code compliant.
Thanks!